examples/lorawan/Makefile

```diff
@@ -24,6 +24,10 @@ USEMODULE += $(DRIVER)
 USEMODULE += fmt
 FEATURES_REQUIRED += periph_rtc
 
+# on EFM32 CPUs RTC and RTT features are using the same peripheral and thus
+# cannot be used at the same time (this application requires both rtc and rtt)
+FEATURES_BLACKLIST += arch_efm32
```

Comment:
Why is a blacklist needed, there should be a `FEATURES_CONFLICT`

Reply:
Apparently it's not taken into account by Murdock which is still trying to build this application for EFM32 boards. There's the same issue with some AVR based boards.

Reply:
See https://github.com/RIOT-OS/RIOT/pull/14753

Reply:
Can you remove this now that #14753 is in?

Reply:
removed